src/js/angular/clustermanagement/templates/cluster-list.html

```diff
@@ -209,7 +209,7 @@
                 <div class="indented-div">
                         <span class="padding-label wrapper-div"
                               gdb-tooltip="{{'cluster_management.cluster_page.advanced_options_tooltip' | translate}}">
-                            <button type="button" class="text-btn" data-toggle="collapse" style="text-align: left"
+                            <button type="button" class="text-btn advanced-options-btn" data-toggle="collapse" style="text-align: left"
```

Comment:
it's minor, but this inline style can be moved in the stylesheet or replace it with the utility class `text-left`